README.md

```diff
@@ -170,7 +182,7 @@ The two other files can be used to vote without parsing the YAML file.
 Encrypting the ballot is necessary to ensure people voting early do not
 interfere or influence folks voting after them. At the end of the vote, the
 the Vote Private Key can be made public, so anyone can decrypt the ballots and verify
-the result themself. Or it can decided that the Vote Private Key won't be shared in
+the result themselves. Or it can decided that the Vote Private Key won't be shared in
```

Comment:
I don't think this is correct, "anyone" would represent a single person, with a single self